lib/territory.js

```diff
@@ -28,3 +29,103 @@ export function nextBillingWithGrace (sub) {
   if (!sub) return null
   return datePivot(new Date(sub.billPaidUntil), { days: TERRITORY_GRACE_DAYS })
 }
+
+function cleanMarkdownText (text) {
+  if (!text) return ''
+
+  let cleaned = removeMd(text)
+  cleaned = cleaned
+    .replace(/\s+/g, ' ') // Replace multiple spaces with single space
+    .replace(/\[\[([^\]]+)\]\]\([^)]+\)/g, '$1') // Remove nested markdown links, keep inner text
+    .replace(/\[([^\]]+)\]\([^)]+\)/g, '$1') // Remove regular markdown links, keep text
+    .replace(/!\[([^\]]*)\]\([^)]+\)/g, '') // Remove images completely
+    .replace(/`([^`]+)`/g, '$1') // Remove code formatting
+    .replace(/\*\*([^*]+)\*\*/g, '$1') // Remove bold formatting
+    .replace(/\*([^*]+)\*/g, '$1') // Remove italic formatting
+    .trim()
+
+  return cleaned
+}
+
+function truncateAtSentenceBoundary (text, maxLength = 160) {
+  if (!text || text.length <= maxLength) return text
+  const sentences = text
+    .split(/[.!?]+/)
+    .map(s => s.trim())
+    .filter(Boolean)
+
+  let result = ''
+  let currentLength = 0
+
+  for (const sentence of sentences) {
+    const separator = result ? '. ' : ''
+    const newLength = currentLength + separator.length + sentence.length
+
+    if (newLength <= maxLength - 3) {
+      result += separator + sentence
+      currentLength = newLength
+    } else {
+      break
+    }
+  }
+  if (result) {
+    return result + '...'
+  }
+  // Fallback to simple truncation
+  return text.substring(0, maxLength - 3) + '...'
+}
+
```

Comment:
why not share the item seo stuff?

if the item seo stuff is lame in some way, improve and share it with territories. I think they should share seo - they are both mostly markdown documents.